Package.swift

```diff
@@ -16,7 +16,7 @@ let package = Package(
         // .package(url: /* package url */, from: "1.0.0"),
         .package(url: "https://github.com/IBM-Swift/BlueSocket.git", .upToNextMinor(from: "0.12.0")),
         .package(url: "https://github.com/IBM-Swift/BlueSSLService.git", .upToNextMinor(from: "0.12.0")),
-        .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .upToNextMinor(from: "0.8.0")),
+        .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .upToNextMinor(from: "1.0.0")),
```

Comment:
You might like to consider loosening this to just `.from("1.0.0")`

Reply:
How about `upToNextMajor`?

Reply:
Same thing, I think.

Reply:
Oh okay. I'll do that. `from` just sounds a bit scary in that it would pick up a new major as well.